projects/ion/src/lib/click-outside.directive.ts

```diff
@@ -22,9 +22,17 @@ export class ClickOutsideDirective {
       return;
     }
     const clickedInside = this.elementRef.nativeElement.contains(targetElement);
-    if (!clickedInside) {
+    if (!clickedInside && !this.isOnDropdown(targetElement)) {
       this.clickOutside.emit(null);
       this.firstOpen = true;
     }
   }
+
+  private isOnDropdown(targetElement: HTMLElement): boolean {
+    return (
+      targetElement &&
+      targetElement.className &&
+      targetElement.className.includes('ant-select-dropdown-menu-item')
```

Comment:
Add the string like a new param to this directive, we should not mock here a class of third.

Reply:
Already working on this